builtin/logical/pki/path_fetch_issuers.go

```diff
@@ -162,6 +179,8 @@ func (b *backend) pathGetIssuer(ctx context.Context, req *logical.Request, data
 			"manual_chain":            respManualChain,
 			"ca_chain":                issuer.CAChain,
 			"leaf_not_after_behavior": issuer.LeafNotAfterBehavior,
+			"use_for_issuance":        issuer.UseForIssuance,
```

Comment:
Rather than bool arguments on the API, would a StringSlice of usage types be both more future proof and a little easier to use?  e.g. usages="issuance,crl_signing".  I don't feel strongly though.

Reply:
Eh, then you've gotta iterate over the usage and order isn't guaranteed so you're stuck doing multiple passes.

I think a map (or a proper set type if Go had one) is a better construct for that, but then you might as well just use bitvectors I guess. :/

Reply:
Ah, no, I see what you mean, definitely.

Reply:
I thought Scott was advocating for this also on the storage entry.